src/UserRepository.js

```diff
@@ -11,6 +13,13 @@ class UserRepository {
       return allAvgSteps += user.dailyStepGoal/this.userData.length
     },0);
   }
+  returnFriendFullName(users) {
+    console.log(users)
+    console.log(this.userData.friends)
```

Comment:
I think we got this kinda in a working spot at this point, we will will need to remove the console.logs